man/systemd.netdev.xml

```diff
@@ -791,6 +791,14 @@
           default will be used.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Independent=</varname></term>
+        <listitem>
+          <para>Takes a boolean. When true, the vxlan interface is created without any underlying network
+          interface. Defaults to false, which means that a .network file that requests this tunnel using
+          <varname>Tunnel=</varname> is required for the tunnel to be created.</para>
```

Comment:
Could you also update `Tunnel=` -> `VXLAN=`?

Reply:
Ugh, `[GENEVE]` section also needs to such copy-and-paste mistake...

Reply:
Hm?? `[GENEVE]` section does not support `Independent=`... Please drop it. Ugh, ugh, ugh...

Reply:
Yeah I was thinking about this aswell. That option for VXLAN came from https://github.com/systemd/systemd/issues/17064. Maybe I should open one for GENEVE aswell? Although I am currently not using it at all.

Reply:
Let's drop the documentation for geneve now. And re-add when it is actually supported.

Reply:
It is not necessary to split PR. Let's also fix that in this PR. Thank you!

Reply:
Assuming with "dropping the doc for geneve" you mean removing the independent doc option on the `[GENEVE]` section, this is already done.

Reply:
Ah, indeed. Thanks.